### PR TITLE
Dịch "rmsprop_vn" - Phần 3

### LINKED ISSUE #1960: Dịch "rmsprop_vn" - Phần 3
Phần cần dịch: https://github.com/aivivn/d2l-vn/blame/master/chapter_optimization/rmsprop_vn.md#L174-L226

### Hướng dẫn
- [Dịch trực tiếp trên website](https://github.com/aivivn/d2l-vn/blob/master/CONTRIBUTING.md).
- Sau khi nhấn vào phần cần dịch ở trên, hãy chọn **Normal view** và làm theo hướng dẫn ở trên để có thể bắt đầu dịch.

### PR DESCRIPTION
**Pull request by davidnvq**
## Pull Request Checklist

**Đánh dấu tick (sau khi "Create Pull Request") vào các mục dưới đây:**

* [x] Pull Request này tương ứng với issue nào? Điền số issue sau dấu `#` (không có dấu cách): Close #1960

* [x] Bản dịch này có bám sát **[Bảng thuật ngữ](https://github.com/aivivn/d2l-vn/blob/master/glossary.md)** không? Nếu một từ/cụm chưa có trong Bảng thuật ngữ, hãy tạo một Pull Request riêng đề xuất cách dịch từ/cụm đó.

* [x] Format của bản gốc có được giữ nguyên không?
     
* [x] Trong một đoạn văn, mỗi câu văn đã được viết trong một dòng, giữa các dòng không có dòng trắng. Markdown sẽ coi những dòng liền nhau không có dòng trắng là một đoạn văn. Việc này giúp công đoạn review được thuận tiện hơn. Bạn đã kiểm tra rằng mỗi đoạn văn đã nằm một dòng?

* [x] Điền tên đầy đủ của bạn vào mục **Những người thực hiện** ở cuối tập tin.

* [ ] Điền tên những reviewers mà bạn thấy có nhiều đóng góp cho Pull Request này. Xem danh sách Contributors **[tại đây](https://github.com/aivivn/d2l-vn/blob/master/docs/contributors_info.md)**.

## Lưu ý cho người phản biện (Reviewer)

1. Chỉ review nếu Pull Request (PR) này đã sẵn sàng cho việc review. Một PR sẵn sàng khi PR đó không có label `status: wip`.
2. Đã đọc kỹ hướng dẫn đóng góp và xem qua **[Bảng thuật ngữ](https://github.com/aivivn/d2l-vn/blob/master/glossary.md)**.

### CODE REVIEW

File: chapter_optimization/rmsprop_vn.md [hunk 1]
```diff
@@ -177,14 +177,15 @@ d2l.train_ch11(rmsprop, init_rmsprop_states(feature_dim),
...
-## *dịch tiêu đề phía trên*
+## Lập trình Súc tích
 
 <!--
 Since RMSProp is a rather popular algorithm it is also available in the `Trainer` instance.
 All we need to do is instantiate it using an algorithm named `rmsprop`, assigning $\gamma$ to the parameter `gamma1`.
 -->
 
-*dịch đoạn phía trên*
+Do RMSProp là thuật toán khá phổ biến, nên nó cũng được tích hợp sẵn sẵn trong đối tượng `Trainer`.
```
Comment (cuongvng) [inline chapter_optimization/rmsprop_vn.md:187]:
```suggestion
Do RMSProp là thuật toán khá phổ biến, nên nó cũng được tích hợp sẵn trong thực thể `Trainer`.
```

File: chapter_optimization/rmsprop_vn.md [hunk 2]
```diff
@@ -221,7 +225,10 @@ d2l.train_gluon_ch11('rmsprop', {'learning_rate': 0.01, 'gamma1': 0.9},
 4. Would you want to adjust $\gamma$ as optimization progresses? How sensitive is RMSProp to this?
 -->
 
-*dịch đoạn phía trên*
+1. Điều gì sẽ xảy ra nếu ta đặt $\gamma = 1$? Giải thích tại sao?
+2. Biến đổi bài toán tối ưu thành cực tiểu hoá $f(\mathbf{x}) = 0.1 (x_1 + x_2)^2 + 2 (x_1 - x_2)^2$. Sự hội tụ sẽ diễn ra như thế nào?
+3. Hãy thử áp dụng RMSProp cho một bài toán học máy cụ thể, chẳng hạn như huấn luyện trên tập Fashion-MNIST. Hãy thí nghiệm với các cách điều chỉnh tốc độ học khác nhau.
+4. Bạn có muốn điều chỉnh $\gamma$ như các tiến bộ tối ưu hóa? Hãy cho biết độ nhạy của RMSProp với điều chỉnh này?
```
Comment (cuongvng) [inline chapter_optimization/rmsprop_vn.md:231]:
```suggestion
4. Bạn có muốn điều chỉnh $\gamma$ trong quá trình tối ưu? Hãy cho biết độ nhạy của RMSProp với điều chỉnh này?
```

Comment (davidnvq) [reply chapter_optimization/rmsprop_vn.md:231]: Nghe dịch `như các tiến bộ` thì hơi ngang trong tiếng Việt :)) nhưng `trong quá trình` thì có vẻ k phản ánh đúng lắm. Maybe

```suggestion
4. Bạn có muốn điều chỉnh $\gamma$ trong thuật toán? Hãy cho biết độ nhạy của RMSProp với điều chỉnh này?
```

Comment (cuongvng) [reply chapter_optimization/rmsprop_vn.md:231]: câu này em hiểu là có muốn điều chỉnh gamma *khi* việc tối ưu tiến triển không, hơi dài nên em rút thành `trong quá trình`. Nếu sợ hiểu sai thì cứ để vậy.
```suggestion
4. Bạn có muốn điều chỉnh $\gamma$ khi việc tối ưu tiến triển không? Hãy cho biết độ nhạy của RMSProp với điều chỉnh này?
```

Comment (davidnvq) [reply chapter_optimization/rmsprop_vn.md:231]: oki, thế chắc đúng hơn. Anh cũng hiểu nhầm nghĩa từ `as`. Tí thì tưởng `progresses` là danh từ :))

File: chapter_optimization/rmsprop_vn.md [hunk 3]
```diff
@@ -177,14 +177,15 @@ d2l.train_ch11(rmsprop, init_rmsprop_states(feature_dim),
...
-## *dịch tiêu đề phía trên*
+## Lập trình Súc tích
 
 <!--
 Since RMSProp is a rather popular algorithm it is also available in the `Trainer` instance.
 All we need to do is instantiate it using an algorithm named `rmsprop`, assigning $\gamma$ to the parameter `gamma1`.
 -->
 
-*dịch đoạn phía trên*
+Do RMSProp là thuật toán khá phổ biến, nên nó cũng được tích hợp sẵn trong thực thể `Trainer`.
```
Comment (minhduc0711) [inline chapter_optimization/rmsprop_vn.md:187]:
```suggestion
Do RMSProp là thuật toán khá phổ biến, nó cũng được tích hợp sẵn trong thực thể `Trainer`.
```

File: chapter_optimization/rmsprop_vn.md [hunk 4]
```diff
@@ -206,7 +207,10 @@ d2l.train_gluon_ch11('rmsprop', {'learning_rate': 0.01, 'gamma1': 0.9},
 * The coefficient $\gamma$ determines how long the history is when adjusting the per-coordinate scale.
 -->
 
-*dịch đoạn phía trên*
+* Thuật toán RMSProp rất giống với Adagrad ở chỗ cả hai đều sử dụng bình phương của gradient để chuyển đổi hệ số.
+* RMSProp có điểm chung với phương pháp động lượng là đều sử dụng trung bình rò rỉ. Tuy nhiên, RMSProp sử dụng kỹ thuật này để điều chỉnh tiền điều kiện theo hệ số.
```
Comment (minhduc0711) [inline chapter_optimization/rmsprop_vn.md:211]:
```suggestion
* RMSProp có điểm chung với phương pháp động lượng là chúng đều sử dụng trung bình rò rỉ. Tuy nhiên, RMSProp sử dụng kỹ thuật này để điều chỉnh tiền điều kiện theo hệ số.
```

File: chapter_optimization/rmsprop_vn.md [hunk 5]
```diff
@@ -221,7 +225,10 @@ d2l.train_gluon_ch11('rmsprop', {'learning_rate': 0.01, 'gamma1': 0.9},
 4. Would you want to adjust $\gamma$ as optimization progresses? How sensitive is RMSProp to this?
 -->
 
-*dịch đoạn phía trên*
+1. Điều gì sẽ xảy ra nếu ta đặt $\gamma = 1$? Giải thích tại sao?
+2. Biến đổi bài toán tối ưu thành cực tiểu hoá $f(\mathbf{x}) = 0.1 (x_1 + x_2)^2 + 2 (x_1 - x_2)^2$. Sự hội tụ sẽ diễn ra như thế nào?
+3. Hãy thử áp dụng RMSProp cho một bài toán học máy cụ thể, chẳng hạn như huấn luyện trên tập Fashion-MNIST. Hãy thí nghiệm với các cách điều chỉnh tốc độ học khác nhau.
+4. Bạn có muốn điều chỉnh $\gamma$ khi việc tối ưu tiến triển không? Hãy cho biết độ nhạy của RMSProp với điều chỉnh này?
```
Comment (minhduc0711) [inline chapter_optimization/rmsprop_vn.md:231]:
```suggestion
4. Bạn có muốn điều chỉnh $\gamma$ khi việc tối ưu tiến triển không? Hãy cho biết độ nhạy của RMSProp với việc điều chỉnh này?
```
